app/components/bulk_upload_error_row_component.html.erb

```diff
@@ -18,16 +20,50 @@
         <% end %>
 
         <%= table.with_body do |body| %>
-          <% bulk_upload_errors.each do |error| %>
+          <% bulk_upload_errors.reject { |error| error.category == "soft_validation" }.each do |error| %>
             <% body.with_row do |row| %>
-              <% row.with_cell(header: true, text: error.cell) %>
-              <% row.with_cell(text: question_for_field(error.field)) %>
-              <% row.with_cell(text: error.error) %>
-              <% row.with_cell(text: error.field.humanize) %>
+                <% row.with_cell(text: error.cell) %>
+                <% row.with_cell(text: question_for_field(error.field)) %>
+                <% row.with_cell(header: true, text: error.error) %>
+                <% row.with_cell(text: error.field.humanize) %>
             <% end %>
           <% end %>
         <% end %>
       <% end %>
     <% end %>
+
+    <% potential_errors = bulk_upload_errors.select { |error| error.category == "soft_validation" } %>
```

Comment:
You could potentially do something like `potential_errors, errors = bulk_upload_errors.partition { |error| error.category == "soft_validation" }` to avoid iterating over bulk_upload_errors twice

Reply:
That's a good idea, will adjust.